pdns/dnsdistdist/dnsdist-configuration-yaml.cc

```diff
@@ -1425,6 +1438,9 @@ std::shared_ptr<DNSSelector> getKeyValueStoreRangeLookupSelector(const KeyValueS
   }
   auto selector = dnsdist::selectors::getKeyValueStoreRangeLookupSelector(kvs, lookupKey);
   return newDNSSelector(std::move(selector), config.name);
+#else
+  throw std::runtime_error("Unable to create KVS action: neitherCDB nor LMDB support are enabled");
```

Comment:
syslog: last message repeated 3 times
```suggestion
  throw std::runtime_error("Unable to create KVS action: neither CDB nor LMDB support are enabled");
```